2022-04-12 17:59:50,284   CRITICAL   client.py             Потеряно соединение с сервером.
2022-04-12 17:59:50,289   DEBUG      threading.py          Вызвана функция: run с параметрами: (<ClientReceiver(Thread-2, started daemon 8580)>,), {}. Результат работы функции: None. Вызов из функции: _bootstrap_inner 
